front/app/services/actionTakingRules.ts

```diff
@@ -177,6 +177,36 @@ export const getIdeaPostingRules = ({
     }
 
     // continuous, not an enabled ideation project
+    // TODO: Will need to update this section after we add new permissions in back office in i5
+    if (
+      isNilOrError(phase) &&
```

Comment:
It feels like we should be abstracting this somehow, but I don't feel like it fits in with the participation method config really? It has less to do with the participation method and more with project attributes and whether it's an auth user. Do you have any other suggestion though? I've just followed the current pattern already used.

Reply:
Yeah agreed - would be nice to fix up the whole file :)  Ok. I'll make a note of it and leave it as-is for now.